### DISCUSSION TITLE
如何使某个实体不受其他实体遮挡，透视效果

### DISCUSSION THREAD

**Discussion by eatfishs** [galacean/discussion#38] [Engine] [answered]
类似 Babylon.js 中的 mesh.renderingGroupId=99;

**Comment by GuoLei1990** [comment, marked as answer]
`Material.renderState.renderQueueType` 可以指定渲染队列类型，比如透明和非透明
`Renderer.priority` 可以指定在队列中的渲染顺序，可以结合 `renderQueueType` 混合使用
